README.md

```diff
@@ -12,6 +12,19 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
 > [!NOTE]
 > If you're a validator and would like to add your data to the API, please open a pull request (PR) with your JSON file, following the structure of `./public/validators/.example` in the `./public/validators` directory.
 
+## Add your validator information
+
+If you run your own validator and want it to be recognised, you can add a name, logo and other publicly available fields. For example, the Nimiq Wallet will use this information to display your validator to users.
+
+In order to do so, follow these steps:
+
+1. [Fork this repository](https://github.com/nimiq/validators-api/fork).
+2. Create a new file in the [validators](./public/validators/) folder. You can use `.example.json` as a reference.
```

Comment:
I had a look at the example.json and not all fields were quite clear to me:
- icon: Should this icon be added to the repo too? Or via a publicly reachable domain?
- fee: How is this field interpreted? Can it be int or decimals? Does a value of 1 mean 1% fee?
- isPool: Why is the default `null`? I would expect a boolean here.

Reply:
Ok, I can change this file to:

```
{
     "icon": "Optional. A base64 encoded SVG icon. It should start with `data:image/svg+xml;base64...`. If you are not sure how to get this value, you can leave this field empty and add the svg in the PR. One of the maintainers will convert it for you :)",
     "fee": 0.1,
     "isPool": false
}
```

I want to keep this field JSON-compatible, so I will move the comments also to this readme

Reply:
Alright 👍